Saudox/app/Http/Controllers/Profissional/ProfissionalController.php

```diff
@@ -170,13 +209,49 @@ public function salvarEditarPaciente(Request $request) {
             'unique' => 'O :attribute já está cadastrado',
         ];
 
+        if($entrada['lista_irmaos'] == "") {
+            $entrada['lista_irmaos'] = "Nenhum";
+        }
+
+        $rota_erro = "profissional.criar_paciente.editar";
+
+
+        if(!isset($entrada["data_nascimento"])
+            || $entrada["data_nascimento"] == ""
+            || $entrada["data_nascimento"] == NULL
+            || !strtotime($entrada["data_nascimento"])
+            || !$this->validatarData($entrada["data_nascimento"])
```

Comment:
O 
```php
!$this->validatarData($entrada["data_nascimento"])
```
já faz vários dos trabalhos a cima, não?

Reply:
não, se for vazio, por exemplo, ele dá 1 primeiro de janeiro de 1970, mas vou mover isso tudo pra dentro da função ainda assim